client/dist/css/related_items.css

```diff
@@ -94,12 +94,20 @@
     background-color: #AA0000;
 }
 
-.category, .productName, .price{
+
+.category, .productName, .price, .stars{
     /* padding: 20px; */
     padding-left:8px
 }
+
 .category {
     font-size: small;
 }
 
+.cardText{
+    /* max-width: 100px ; */
+    max-height: 20px;
+}
+
```

Comment:
This file looks good, maybe in the future we can see to finding static percentages for all of our components to use so we can have consistent div sizes